metals/src/main/scala/scala/meta/internal/implementation/ImplementationProvider.scala

```diff
@@ -177,13 +212,18 @@ final class ImplementationProvider(
   private def methodInParentSignature(
       sig: ClassSignature,
       childInfo: SymbolInformation,
+      globalSymbolTable: Option[GlobalSymbolTable],
       childASF: Map[String, String] = Map.empty
-  ): Seq[Location] = {
+  ): Seq[Either[Location, SymbolInformation]] = {
     sig.parents.flatMap {
-      case parentSym: TypeRef =>
+      case parentSym: TypeRef if parentSym.symbol != "scala/AnyRef#" =>
```

Comment:
Why do we need this if?

Reply:
We get AnyRef as parents, it makes sense not to analyze it